### DISCUSSION TITLE
Keep JPEG XL in bromite moving forward

### DISCUSSION THREAD

**Discussion by earthsound** [bromite/bromite#2594] [Ideas]
I opened a [feature request issue](https://github.com/bromite/bromite/issues/2593), but it was closed due to it not being related to privacy. I'm copying the OP here to spur some discussion into (hopefully) restoring/maintaing JPEG XL support in bromite.

OP:

### Is there a patch available for this feature somewhere?

Yes. It appears the libjxl team (who maintains the [reference implementation of JPEG XL](https://github.com/libjxl/libjxl)) can/is going to maintain up-to-date rebased chromium patches that restore/keep jxl/chromium integration after M109 (when Google removed it). 

Please see the discussion in [Thorium issue #104](https://github.com/Alex313031/Thorium/issues/104) and the [Thorium libJXL repo](https://github.com/Alex313031/thorium-libjxl) created as a submodule to the main Thorium repo designed to restore/maintain JPEG XL support in Thorium/chromium post M109.

### Describe the solution you would like

Although it's not related to privacy, please consider this feature request.

Since Google [removed JPEG XL beginning with M110](https://bugs.chromium.org/p/chromium/issues/detail?id=1178058#c84), I would appreciate being able to keep JPEG XL functionality in bromite when it's updated next. The discussion above and repo are excellent starting points to catch up on how this could be maintained within bromite (and any chromium based browser) by piggy backing upon the work done by the libjxl team and the Thorium devs.

### Describe alternatives you have considered

There isn't really a complete alternative, as Firefox support for JPEG XL doesn't include HDR, unlike (past versions of) vanilla chromium. 

Currently, Jon Sneyers of the libjxl team has stated that [Thorium is the only browser with full JPEG XL support](https://twitter.com/jonsneyers/status/1626878007091527680), and it's enabled by default. 

Thorium has made it an [official goal to keep libjxl post M109](https://github.com/Alex313031/Thorium/issues/104#issuecomment-1430573061).

**Comment by G2G2G2G** [comment]
I agree, I'd like jpeg xl

virtually every firefox fork has jxl enabled by default, 
for others there is https://github.com/zamfofex/jxl-crx too
